src/components/EmailSubscription/index.view.tsx

```diff
@@ -14,6 +14,8 @@ const states = {
   ERRORED: 3,
 };
 const wedgewood = "#4f728e";
+const wedgewood_green = "#5B8F49";
+const wedgewood_red = "#8F3E3B";
```

Comment:
`wedgewood` is actually the color of our background like the bluish color haha. I'd say the red could be like a `mahogany` and green could be a `grassGreen` 😂

Reply:
Aha, I know - it was just mildly amusing. Fixed